pycbc/waveform/ringdown.py

```diff
@@ -882,32 +930,60 @@ def get_fd_from_final_mass_spin(template=None, **kwargs):
     template : object
         An object that has attached properties. This can be used to substitute
         for keyword arguments. A common example would be a row in an xml table.
-    distance : {None, float}, optional
-        Luminosity distance of the system. If specified, the returned ringdown
-        will include the Kerr factor (final_mass/distance).
     final_mass : float
         Mass of the final black hole in solar masses.
     final_spin : float
         Dimensionless spin of the final black hole.
+    distance : {None, float}, optional
+        Luminosity distance of the system. If specified, the returned ringdown
+        will include the Kerr factor (final_mass/distance).
     lmns : list
-        Desired lmn modes as strings (lm modes available: 22, 21, 33, 44, 55).
+        Desired lmn modes as strings. All modes up to l = m = 7 are available.
         The n specifies the number of overtones desired for the corresponding
-        lm pair (maximum n=8).
-        Example: lmns = ['223','331'] are the modes 220, 221, 222, and 330
-    amp220 : float
-        Amplitude of the fundamental 220 mode.  Always required, even if 220
-        mode has not been selected. Note that if distance is given,
-        this parameter will have a completely different order of magnitude.
-        See table II in https://arxiv.org/abs/1107.0854 for an estimate.
+        lm pair, not the overtone number; maximum n=8. Example:
+        lmns = ['223','331'] are the modes 220, 221, 222, and 330
+    ref_amp : str, optional
```

Comment:
Why is this needed?

Reply:
Oh, I see in the function `lm_amps_phases` that it defaults to amp220 if no ref_amp is given, so I guess it is ok